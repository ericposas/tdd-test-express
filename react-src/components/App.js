import React, { Component } from 'react'
import { HashRouter as Router, Route, Switch } from 'react-router-dom'
import Home from './Home'

export default function App(props) {

  return (
    <>
      <Router>
          <Switch>
            <Route exact path='/'>
              <Home name='Hello.'/>
            </Route>
          </Switch>
      </Router>
    </>
  )

}
