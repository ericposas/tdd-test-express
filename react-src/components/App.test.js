import React from 'react'
import { shallow, mount } from 'enzyme'
import App from './App'
import { HashRouter as Router, Switch } from 'react-router-dom'

test('component should render', () => {
  const wrapper = shallow(<App />)
  expect(wrapper).toMatchSnapshot()
  wrapper.unmount()
})
