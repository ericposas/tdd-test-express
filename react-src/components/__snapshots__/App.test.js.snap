// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component should render 1`] = `
<Fragment>
  <HashRouter>
    <Switch>
      <Route
        exact={true}
        path="/"
      >
        <Home
          name="Hello."
        />
      </Route>
    </Switch>
  </HashRouter>
</Fragment>
`;
